### DISCUSSION TITLE
Github action trigger with an ArgoCD if SSO enabled

### DISCUSSION THREAD

**Discussion by shkelzeen** [omegion/argocd-actions#20] [Q&A] [answered]
Hi 

I am trying to use this github action to sync my project but I think the actions is unable to trigger the sync since our ArgoCD has sso enabled. Can that be a problem do we have any information and if yes is there any workaround?

**Comment by omegion** [comment, marked as answer]
Hi, can you give more information about the error you're having, please?

**Reply by shkelzeen** [reply]
I found the issue it was on my side, I tried to use user token which is generated when login in on local but I need a different type, I need to create a token into argocd instance. You can close this  this question .
